src/rTerminal.ts

```diff
@@ -116,7 +116,7 @@ export async function runTextInTerm(term: Terminal, text: string) {
         term.sendText(text);
     } else {
         for (const line of text.split('\n')) {
-            await delay(8); // Increase delay if RTerm can't handle speed.
+            await delay(config().get('rtermSendDelay')); // Increase delay if RTerm can't handle speed.
```

Comment:
I'm not sure if it is better that we get the value of `rtermSendDelay` before the `for` loop and apply the same number in each iteration, or constantly fetch the value so that it is immediately effective even when user is in the middle of sending a big chunk of code?

Reply:
Good point, I think to me it would be unintuitive for the delay speed to change after having submitted the code. I think it also makes sense to not do repeated calls to get the config variable so I have pulled it out to its own variable prior to the for loop